docs/catalog.md

```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
   }
   ```
 
+### Using an action sequence and a change trigger to get a document from a Cloudant database
```

Comment:
I had problems coming with a good title. yeah I think s/get/process will do the trick